components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -17,9 +17,6 @@ module omero {
 
     module api {
 
-        /**
-         * See <a href="http://hudson.openmicroscopy.org.uk/job/OMERO/javadoc/ome/api/IAdmin.html">IAdmin.html</a>
```

Comment:
Is there a URL we can redirect these to until http://trac.openmicroscopy.org.uk/ome/ticket/2101 is tackled?